src/algebra/ordered_ring.lean

```diff
@@ -833,11 +827,11 @@ have a * a ≤ (0 : α), from calc
        ... = 0             : h,
 eq_zero_of_mul_self_eq_zero (le_antisymm this (mul_self_nonneg a))
 
--- The proof doesn't need commutativity but we have no `decidable_linear_ordered_ring`
+-- The proof doesn't need commutativity but we have no `linear_ordered_ring`
```

Comment:
Is this statement no longer true?  (i.e., `linear_ordered_ring` does exist, so the lemma can now be stated in that context?)

Reply:
Moved this and a few other lemmas to `linear_ordered_ring`. Could you please have a look at the other lemmas about `abs` in this file and move (some of) them to `linear_ordered_ring` too?

Reply:
Yes, I can do this.  You are editing that file now, right?  Let me know when you finish.

Reply:
I'm done.

Reply:
I did this (well-spotted!). I also moved a couple of lemmas from `linear_ordered_ring` to `linear_ordered_add_comm_group`.